docs/Lua API.rst

```diff
@@ -4415,12 +4415,29 @@ General script API
   command-line. The ``name`` argument should be the name of the script without
   its extension, as would be used on the command line.
 
-  Note that this function lets Lua errors propagate to the caller.
+  Example::
+     dfhack.run_script("armoks-blessing", "all")
+  Scipts that use a dash(``-``) in their arguments still require one when called
+  from ``dfhack.run_script``.
+  Example::
+     dfhack.run_script("gaydar", "-all")
+
+  Unlike ``dfhack.run_command``, ``dfhack.run_script`` allows a different
+  different script to be executed and its value passed as an argument to the main
+  function called by ``dfhack.run_script``.
+
+  Example::
+      dfhack.run_script("some_script", dfhack.gui.getDwarfmodeViewDims());
+
+  While this implmenation is currently a functional, its purpose and stability
+  are limited. As such, ``dfhack.reqscript(name)`` should be used.
```

Comment:
This isn't accurate. `run_script` *runs* a script, which is useful and stable (it is used every time any user runs a script). `reqscript` *imports* a script. Perhaps we add a cross-reference to `reqscript` here, but it is already in the "Importing Scripts" section immediately below this one.